antsibull/build_ansible_commands.py

```diff
@@ -290,6 +307,7 @@ async def make_collection_dist(name, version, package_dir, dest_dir):
     loop = asyncio.get_running_loop()
 
     # Create the python sdist
+    # pyre-ignore[16], pylint:disable=no-member
```

Comment:
Nice!  I didn't know you could combine these comments on the same line.

Reply:
I didn't knew either, I tried and it happened to work :)